ros/ros

```diff
@@ -72,6 +72,28 @@ GitCommit: bca53bf4c09d771be3ff735da4157203b53ebc2b
 Directory: ros/iron/ubuntu/jammy/perception
 
 
+################################################################################
+# Release: jazzy
+
+########################################
+# Distro: ubuntu:noble
+
+Tags: jazzy-ros-core, jazzy-ros-core-noble
+Architectures: amd64, arm64v8
+GitCommit: 543aabeed37ee905ac3737fd38ffbc9894e997db
+Directory: ros/jazzy/ubuntu/noble/ros-core
+
+Tags: jazzy-ros-base, jazzy-ros-base-noble, jazzy
+Architectures: amd64, arm64v8
+GitCommit: 543aabeed37ee905ac3737fd38ffbc9894e997db
+Directory: ros/jazzy/ubuntu/noble/ros-base
+
+Tags: jazzy-perception, jazzy-perception-noble
+Architectures: amd64, arm64v8
+GitCommit: 543aabeed37ee905ac3737fd38ffbc9894e997db
+Directory: ros/jazzy/ubuntu/noble/perception
```

Comment:
I can confirm [`desktop` works](https://github.com/sloretz/ros_oci_images?tab=readme-ov-file#jazzy-pre-release-images) from the ros2-testing apt repo, and it looks like `desktop_full` is should work now too.

https://repo.ros2.org/status_page/ros_jazzy_default.html?q=variant

Reply:
Thanks for the heads up! It was not the case when I initially opened this.
Then I'll add it to ci here so that we get the automated build / push of these images as well

Reply:
It's not clear to me what is now the right place to report this type of issue.
What would you recommend @sloretz ? ros/rosdistro ?

Reply:
> It's not clear to me what is now the right place to report this type of issue.
> What would you recommend @sloretz ? ros/rosdistro ?

That or https://github.com/ros2/ros2

@clalancette FYI Seems like there might be an install issue with `ros-jazzy-simulation`.

Reply:
Yeah, we have to land https://github.com/ros2/variants/pull/42 , and then re-release.  That should unblock this.

Reply:
@clalancette says

> We need to land https://github.com/ros2/variants/pull/42 and https://github.com/ros-infrastructure/rep/pull/401

Sounds like `desktop` works, but `simulation` and `desktop_full` are not yet installable

Reply:
:+1:

PR following the new release of the variants for jazzy: https://github.com/osrf/docker_images/pull/742